### PR TITLE
Updated login styles from B

### PR DESCRIPTION
**Pull request by ChadWCox**
Updated some minor styling adjustments for login and register from team B repo

### CODE REVIEW

**Review by katieolson84 — APPROVED** [review]
Approving since we went over these changes together.

File: src/components/pages/Login/LoginContainer.js [hunk 1]
```diff
@@ -62,9 +62,12 @@ const LoginContainer = () => {
...
         <div className="login-page">
           <div id="sign-in-widget" aria-label="login form" />
-          <Link className="link-styles" to="/signup">
-            <p>Don't have an account? Register here</p>
-          </Link>
+          <p className="register">
+            Don't have an account?{' '}
+            <Link className="link-styles" to="/signup">
+              <span>Register here</span>
+            </Link>
```
Comment (paulstgermain) [inline src/components/pages/Login/LoginContainer.js:69]: Very nice, I like the way our link is separated to include just the 'Register Here' call to action - this'll look even nicer than what we had!

**Review by paulstgermain — APPROVED** [review]
Approving changes on the basis of all conditions for PRs met as well as it being a logical, concise change.

**Review by dgamboa — COMMENTED** [review]
Looks great, all of the key parts of B seem to have made it over.

**Review by nwestallen — APPROVED** [review]
These changes look good to me

**Pull request merged**